src/ScottPlot4/ScottPlot/Drawing/GDI.cs

```diff
@@ -36,14 +36,20 @@ public static SizeF MeasureString(string text, Font font)
             using (Bitmap bmp = new Bitmap(1, 1))
             using (Graphics gfx = Graphics(bmp, lowQuality: true))
             {
-                return MeasureString(gfx, text, font.Name, font.Size, font.Bold);
+                return MeasureString(gfx, text, font.Family, font.Size, font.Bold);
             }
         }
 
         public static SizeF MeasureString(Graphics gfx, string text, string fontName, double fontSize, bool bold = false)
+        {
+            var fontFamily = InstalledFont.ValidFontFamily(fontName);
+            return MeasureString(gfx, text, fontFamily, fontSize, bold);
+        }
+
+        public static SizeF MeasureString(Graphics gfx, string text, FontFamily fontFamily, double fontSize, bool bold = false)
         {
             var fontStyle = (bold) ? FontStyle.Bold : FontStyle.Regular;
-            using (var font = new System.Drawing.Font(fontName, (float)fontSize, fontStyle, GraphicsUnit.Pixel))
+            using (var font = new System.Drawing.Font(fontFamily, (float)fontSize, fontStyle, GraphicsUnit.Pixel))
```

Comment:
I'm tempted to explore if it would be worth it to cache the combination of font $(family, size, style)$ tuples, rather than creating a new GDI font every time, in my (anecdotal, not empirical) experience string measurement is a relatively common occurrence. In theory, this could even be a global cache, not just for string measurement.

Unfortunately, GDI objects are not thread-safe (discussed in #978), and while there are solutions to that, they're undoubtedly not worth it. Especially when we're supposed to be ditching System.Drawing.

So I'm mostly just saying this for a later time when we're using a different graphics library where they either are thread-safe, or we're willing to invest in the effort to use them safely.

Reply:
I'm focusing on allocations and in my case there are bigger problems before `System.Drawing.Font`.
This is the summary of the repro in the first post.

|Type|Allocations|% of total|
|-|-|-|
|System.Drawing.FontFamily|216840|36.23|
|System.String|175106|29.26|
|System.Double[]|35915|6.00|
|System.Double|15607|2.61|
|System.Func<,>|15028|2.51|
|System.Collections.Generic.List<>|10453|1.75|
|System.Linq.Enumerable.WhereListIterator<>|9986|1.67|
|ScottPlot.Plottable.SignalPlotBase<System.Double>.IntervalMinMax.<GetPoints>d__4|7700|1.29|
|ScottPlot.Plottable.SignalPlotBase<System.Double>.IntervalMinMax|7700|1.29|
|System.SByte[]|6753|1.13|
|ScottPlot.Renderable.Axis[]|5707|0.95|
|System.Drawing.Graphics|4475|0.75|
|System.Drawing.Font|4287|0.72|
|...|...|...|
|Total|~598k|100|

Reply:
`Double` and `Double[]` are my next target.